### ISSUE TITLE
dataset download

### ISSUE THREAD

**Issue by WangLiguo-kyrie** [PointSite/PointSite#1]
I don't know how to get the authority of training/test set in google cloud

**Comment by fses91** [comment]
Yes access to the data would be nice.

**Comment by L40S38** [comment]
Hello.
I can't get the dataset from the URLs on README.md, either.
Also, I would have source code to construct a training dataset from scPDB and an evaluation dataset from B277, DT198 and etc. with sequence identity and duplication checked, as well as code to measure performance using atom-IoU and PSP from FPPocket.
